resources/views/components/button.blade.php

```diff
@@ -29,7 +29,7 @@ class="w-4 h-4"
             @if (preg_replace('/[^a-zA-Z]+/', '', $spinner))
                 wire:target="{{ $spinner }}"
             @endif
-            wire:loading>
+            wire:loading.delay>
```

Comment:
I would like to pass the delay modifier, e.g.
`<x-button ... loading-delay="shortest">`

```suggestion
            wire:loading.delay{{ $loadingDelay ? ".{$loadingDelay}":'' }}>
```

Like these examples, but you are free to use any way

Reply:
Yes. I have added the loadingDelay to the BaseButton file 👍🏽